cmd/quotas.go

```diff
@@ -130,11 +133,81 @@ var quotasCmd = &cobra.Command{
 	},
 }
 
+var defaultResources = corev1.ResourceList{
```

Comment:
You hard-coded the variables below because you needed the units, but why not just use the schema

```
defaultResources = {
    "requests.cpu": {
       "quantity": int,
       "unit": func (n: int) <RETURN_TYPE> { return *resource.NewQuantity(n, resource.DecimalSI), }
    },
   // etc
}
```

It would be more elegant not to hardcode everything twice.

Reply:
That is so clean!

Reply:
Maps in golang are a bit different. They don't allow functions to be passed as values and all values must be the same data type (the schema has a function, considered the return type and an int). This would've worked if we had one key, value pair in the map. Struct is an alternative to hold attributes of different data types but you're unable to pass a function (have not seen any examples online). We can chat about this.